tests/sync_observation_period_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import psycopg2
+
+from pedsnetdcc.utils import make_conn_str
+from pedsnetdcc.sync_observation_period import sync_observation_period
+
+
+class SyncObservationPeriodTest(unittest.TestCase):
+
+    def test_observation_period_count(self):
+
+        conn_str = make_conn_str('postgresql://localhost/tmp',
```

Comment:
You could make this test conditional on the existence of an environment variable containing the connection URL, and if the variable doesn't exist, use self.skipTest() ....